lms/djangoapps/instructor_task/api.py

```diff
@@ -337,7 +337,7 @@ def submit_calculate_students_features_csv(request, course_key, features):
     """
     task_type = 'profile_info_csv'
     task_class = calculate_students_features_csv
-    task_input = {'features': features}
+    task_input = features
```

Comment:
@dianakhuang @adampalay Yes, this is a band aid fix. If additional fields need to be added to the profile data report we will have the same problem. The long term fix is lower priority (and may not ever actually be needed) than this fix right now.